### DISCUSSION TITLE
[V3] How to render Components only on Client?

### DISCUSSION THREAD

**Discussion by Shwaolin** [nuxt/nuxt#17552] [Questions] [answered]
Hey, I can't figure out how to fix the following issue:  
![image](https://user-images.githubusercontent.com/72451947/169800767-c05a4c86-e4c1-446a-be00-b01cc34f13d4.png)
![image](https://user-images.githubusercontent.com/72451947/169800819-bc291949-48bc-4df8-8327-5154a777e593.png)
Leads to the following error in the "npm-run"-console and white pages when clicking on `NuxtLink`s  
```[Vue warn]: Failed to resolve component: Editor
If this is a native custom element, make sure to exclude it from component resolution via compilerOptions.isCustomElement.```

**Comment by itchwoot** [comment, marked as answer]
You can wrap it in the client-only component
https://nuxtjs.org/docs/features/nuxt-components/#the-client-only-component

```html
<client-only>
    <div class="flex">
        <Editor id="notes" v-model="user.notes" editor-style="height: 320px" />
    </div>
</client-only>
```

**Reply by Shwaolin** [reply]
Ahh sorry forgot to mention that I'm on v3 (updated the title)  
Or does this component also exist there?

**Reply by TheAlexLichter** [reply]
It also [exists in Nuxt3](https://v3.nuxtjs.org/guide/directory-structure/components#clientonly-component) ☺️
